packages/framework/agent-scheduler/src/scheduler.ts

```diff
@@ -420,8 +422,11 @@ export class AgentSchedulerFactory implements IFluidDataStoreFactory {
 
     public static async createChildInstance(parentContext: IFluidDataStoreContext): Promise<AgentScheduler> {
         const packagePath = [...parentContext.packagePath, AgentSchedulerFactory.type];
-        const router = await parentContext.containerRuntime.createDataStore(packagePath);
-        return requestFluidObject<AgentScheduler>(router, "/");
+        const dataStore = await parentContext.containerRuntime.createDataStore(packagePath);
+        const maybeHandle: FluidObject<IFluidHandle> = (dataStore as any);
+        const handle = await maybeHandle.IFluidHandle?.get();
```

Comment:
this var could be named better. you are getting a handle from a handle, you are getting that object to handle points to, which in this case is the agent scheduler

Reply:
Renamed to `scheduler`, coming in next push.